corehq/apps/userreports/ui/forms.py

```diff
@@ -126,6 +130,13 @@ def clean(self):
 
     def save(self, commit=False):
         self.instance.report_meta.edited_manually = True
+        if toggles.AGGREGATE_UCRS.enabled(self.instance.domain):
+            # hack - if the ID looks like a guid, assume it is non-aggregate, else aggregate
```

Comment:
What do the aggregate data sources ids look like? Is it just an integer?

Reply:
(merging but happy to improve this if you have suggestions)